### DISCUSSION TITLE
Process for requesting addtition of new icons

### DISCUSSION THREAD

**Discussion by kubijo** [carbon-design-system/carbon#14940] [Help] [answered]
We are slowly migrating our app icons from manually processed SVGs to the React component icons included in the CDS library, but it sometimes happens that none of the included icons match the needs of the application...

All that said, what is the process to request addition of new icons into the CDS library?

The last specific case of missing icon was one for `fan`, where I had to be sent back to FontAwesome :).

**Comment by tay1orjones** [comment, marked as answer]
Hi @kubijo, you can find instructions on how to create and contribute [UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute), [app icons](https://www.ibm.com/design/language/iconography/app-icons/production), and [pictograms](https://www.ibm.com/design/language/iconography/pictograms/contribute) on the IBM Design Language site. Thanks!
